tests/interfaces/test_tensorflow.py

```diff
@@ -677,6 +693,8 @@ def test_sampling(self, execute_kwargs):
         """Test sampling works as expected"""
         if execute_kwargs["gradient_fn"] == "device" and execute_kwargs["mode"] == "forward":
             pytest.skip("Adjoint differentiation does not support samples")
+        if execute_kwargs["interface"] == "auto":
+            pytest.skip("Can't detect interface without a non-parametrized tape")

```

Comment:
The suggestion on rewording applies to other occurrences too.
```suggestion
            pytest.skip("Can't detect interface without a parametrized gate in the tape")
```

Reply:
Fixed!